transfer_learning.py

```diff
@@ -151,11 +158,18 @@ def training(padding_value,
   print(table)
   print(f'Sum of trainable parameters: {t_params}')
 
-  model_final = util.transfer_learning.transfer_learning(working_dir, preproc_dataset_name, files_list_, initial_lr,
+  if not only_sample:
+    model_final = util.transfer_learning.transfer_learning(working_dir, preproc_dataset_name, files_list_, initial_lr,
                                            n_epochs, train_ratio, batch_size, model, optimiser, scheduler,
                                            marginal_prob_std_fn, padding_value, transfer_learning_series=transfer_learning_series,
                                            device=device, mask_diff=mask_diff, postfix_=postfix_, 
-                                           serialized_model=serialized_model, cp_chunks=cp_chunks)
+                                           serialized_model=serialized_model, cp_chunks=cp_chunks, continue_=continue_)
+  else:
+    model_final_name = os.path.join(working_dir, 'record_' + postfix_, 'training', 'tanh_transform_threshold' + str(transfer_learning_series[-1]), "ckpt_tmp_{}.pth".format(n_epochs[-1]-1))
```

Comment:
Hard-coded naming convention 'tanh_transform_threshold'. Is this always a reasonable naming convention for the model in the transfer-learning case?